### PR TITLE
feat: add button animation

### PR DESCRIPTION
**Pull request by Aaasisss**
https://github.com/user-attachments/assets/b15e693f-c792-4909-a8aa-bbc6b15ad2bd

### CODE REVIEW

File: src/modules/home/joinXplorers.tsx [hunk 1]
```diff
@@ -1,13 +1,26 @@
+"use client";
```
Comment (20was) [inline src/modules/home/joinXplorers.tsx:1]: Can be excluded ?

Comment (Aaasisss) [reply src/modules/home/joinXplorers.tsx]: done ✅

File: src/modules/home/joinXplorers.module.css
[the remark this answers is not in the document]
Comment (Aaasisss) [reply src/modules/home/joinXplorers.module.css:1]: could not figure out how to apply animations with tailwind da :(

File: src/constants.ts [hunk 2]
```diff
@@ -0,0 +1,2 @@
+export const joinSlackUrl =
+  "https://xplorers-io.slack.com/join/shared_invite/zt-27cdb5ouu-taJbzO_Z0E5kbdMQ8nAJ9A";
```
[the remark this answers is not in the document]
Comment (Aaasisss) [reply src/constants.ts:2]: could not find that document

[the remark this answers is not in the document]
Comment (sonam-serchan) [reply src/constants.ts:2]: Any idea about this @DipeshGainju? https://github.com/orgs/xplorer-io/projects/5?pane=issue&itemId=64898555

If not, all good - we can fix it later!

File: src/modules/home/joinXplorers.module.css
[the remark this answers is not in the document]
Comment (sonam-serchan) [reply src/modules/home/joinXplorers.module.css:1]: ahh all good, not a biggie! we can migrate it to use tailwind later!

**Review by sonam-serchan — DISMISSED** [review]

File: src/constants.ts [hunk 3]
```diff
@@ -0,0 +1,5 @@
...
```
Comment (sonam-serchan) [inline src/constants.ts:2]: Do we still need this Ashis?

**Review by sonam-serchan — APPROVED** [review]
Looks good, thanks for setting up the google form Ashish!

File: src/constants.ts (hunk 3 above)
Comment (Aaasisss) [reply src/constants.ts:2]: We don't really need it da. Just left it there, just in case 😄
I'll remove it then

**Review by sonam-serchan — APPROVED** [review]

**Pull request merged**